docs/source/nlp/joint_intent_slot.rst

```diff
@@ -97,14 +117,14 @@ Model Training
 This is a pretrained Bert based model with 2 linear classifier heads on the top of it,
 one for classifying an intent of the query and another for classifying slots for each token of the query.
 This model is trained with the combined loss function on the Intent and Slot classification task on the given dataset.
-The model architecture is based on the paper: https://arxiv.org/pdf/1902.10909.pdf
+The model architecture is based on the paper `BERT for Joint Intent Classification and Slot Filling <https://arxiv.org/pdf/1902.10909.pdf>`__.
```

Comment:
Could you please add this to the list of references?

Reply:
Added